index.js

```diff
@@ -1,5 +1,4 @@
-'use strict';
```

Comment:
I'm not sure why lint-staged is removing this as part of the `eslint --fix` step in my local environment.

Reply:
you probably have an `eslintrc` file in `$HOME` or something. Try adding `root: true` in the `eslintrc.js` in this repo

Reply:
Thanks for the tip! I had an unexpected `.eslintrc` file one level up from this repo on my machine. Removed that and was able to see my local linting match up with what Travis CI is reporting. 👍